releases.json

```diff
@@ -835,6 +835,17 @@
       "2.2.2-1"
     ]
   },
+  "libglvnd": {
+    "dependency_names": [
+      "gl",
+      "opengl",
+      "glesv1",
+      "glesv2"
```

Comment:
You're missing `GLX` and `EGL` as well

Reply:
Not following here:

```
mangix@ryzen ~/d/w/s/libglvnd-v1.4.0 (gl)> rg declare_de
src/util/meson.build
33:idep_glvnd_pthread = declare_dependency(
45:idep_app_error_check = declare_dependency(
56:idep_utils_misc = declare_dependency(
67:idep_trace = declare_dependency(
81:idep_winsys_dispatch = declare_dependency(
92:idep_cjson = declare_dependency(

src/GLdispatch/meson.build
50:idep_gldispatch = declare_dependency(

src/GLX/meson.build
55:idep_glx = declare_dependency(

src/GLdispatch/vnd-glapi/meson.build
93:  _dep = declare_dependency(
```

the last one is a loop listing ['gl', 'opengl', 'glesv1', 'glesv2']

Reply:
Ooh, that's interesting, `idep_glx` should be publc, and we should have a dependency for `EGL` too (we as in libpkgconf)

Reply:
And, those `idep_*` are short for `internaldep` those aren't for public consumption. I'll send a patch to fix upstream to be consumable